tests/infrastructure/sap/test_sap_hana_vm.py

```diff
@@ -408,12 +408,6 @@ def sap_hana_node(schedulable_nodes):
         return hana_nodes[0]
 
 
-@pytest.fixture(scope="class")
-def skip_if_not_hana_cluster(skip_if_no_cpumanager_workers, sap_hana_node):
-    if not sap_hana_node:
-        pytest.skip(f"No node is marked with sap label {WORKLOAD_NODE_LABEL_NAME} = {WORKLOAD_NODE_LABEL_VALUE}")
```

Comment:
I would suggest to keep verification but fail testrun instead of skipping

Reply:
I agree.